include/cantera/thermo/Phase.h

```diff
@@ -298,6 +298,25 @@ class Phase
     //! (such as "TD", "TP", "SV").
     virtual vector<string> partialStates() const;
 
+    //! Get the size of the vector defining the "partial" state of the phase
+    //! (excluding composition). Used by savePartialState() and restorePartialState()
```

Comment:
I think this comment can just be declarative without parentheses:


```suggestion
    //! Get the size of the partial state vector of the phase. The partial state vector excludes phase composition.
//! Used by savePartialState() and restorePartialState()
```